onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorQAttention.cpp

```diff
@@ -484,7 +696,7 @@ class DmlOperatorQAttention : public DmlOperator
             queryKeyTransposedToMhaEdge.FromNodeIndex = queryKeyTransposedNodeIndex;
             queryKeyTransposedToMhaEdge.FromNodeOutputIndex = 0;
             queryKeyTransposedToMhaEdge.ToNodeIndex = mhaNodeIndex;
-            queryKeyTransposedToMhaEdge.ToNodeInputIndex = mhaStackedQueryKeyIndex;
+            queryKeyTransposedToMhaEdge.ToNodeInputIndex = MhaInputIndex::mhaStackedQueryKeyIndex;
```

Comment:
For consistency we should keep the same style as the other enums or add the prefix to all enums here. Usually for local enums we omit the prefix since it's already obvious what the index is for (the prefix is `mha` and the suffix is `Index`).

Same comment for the other places where it was changed.

Reply:
I will remove enum, the rename was to better point it to an operator in the graph.
But having enum calling out MHA along with each element might be too extra